TVShows/TVShows/Modules/Login/LoginViewController.swift

```diff
@@ -0,0 +1,99 @@
+
+    // MARK: - Public properties -
+
+    var presenter: LoginPresenterInterface!
+
+    @IBOutlet private weak var emailTextField: UITextField!
+    @IBOutlet private weak var passwordTextField: UITextField!
+
+    @IBOutlet private weak var rememberMeButton: UIButton!
+    @IBOutlet private weak var loginButton: UIButton!
+    @IBOutlet private weak var createAnAccountButton: UIButton!
+
+    @IBOutlet private weak var scrollView: UIScrollView!
+    // MARK: - Lifecycle -
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.setNavigationBarHidden(true, animated: animated)
+        _registerForKeyboardNotifications()
+    }
+
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(animated)
+    }
+
+    @IBAction func _loginButtonActionHandler(_ sender: Any) {
+        presenter.didSelectLoginAction(with: emailTextField.text, password: passwordTextField.text)
+    }
+
+    @IBAction func _createAnAccountActionHandler(_ sender: Any) {
+        presenter.didSelectCreateAnAccountAction(with: emailTextField.text, password: passwordTextField.text)
+    }
+    @IBAction func _rememberMeButtonActionHandler(_ sender: Any) {
+        presenter.didPressRememberMe()
+    }
+
+    private func _registerForKeyboardNotifications() {
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(keyboardWillShow),
+                                               name: UIResponder.keyboardWillShowNotification,
+                                               object: nil)
+
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(keyboardWillHide),
+                                               name: UIResponder.keyboardWillHideNotification,
+                                               object: nil)
+    }
+
+    // MARK: - @objc functions
+    @objc func keyboardWillShow(notification: NSNotification) {
+        //give room at the bottom of the scroll view, so it doesn't cover up anything the user needs to tap
+        var userInfo = notification.userInfo!
+        var keyboardFrame:CGRect = (userInfo[UIResponder.keyboardFrameBeginUserInfoKey] as! NSValue).cgRectValue
+        keyboardFrame = self.view.convert(keyboardFrame, from: nil)
+
+        var contentInset:UIEdgeInsets = self.scrollView.contentInset
+        contentInset.bottom = keyboardFrame.size.height
+        scrollView.contentInset = contentInset
+    }
+
+    @objc func keyboardWillHide(notification: NSNotification) {
+        let contentInset:UIEdgeInsets = UIEdgeInsets.zero
+        scrollView.contentInset = contentInset
+    }
+    
+}
+
+// MARK: - Extensions -
+
+extension LoginViewController: LoginViewInterface {
+    func setRememberState(state: Bool) {
```

Comment:
Or you can set one image for normal state, and one for selected (via storyboard) and just toggle `selected` property on `UIButton`